RLP-62952 update-azure-repos-multiple-users

anaghapamidi opened the pull request:
Updated the article with the capability of Azure Repos multiple users.

anaghapamidi committed:
RLP-62952 update-azure-repos-multiple-users

GiladMark commented on code-security/admin_guide/get-started/connect-your-repositories/add-azurerepos.adoc:
```diff
@@ -3,37 +3,84 @@
+Integrating Prisma Cloud with Azure Repos makes it possible for Cloud Code Security to scan your Infrastructure-as-code files (Terraform and CloudFormation) and monitor configuration issues in development.
 As a prerequisite you are required to add the Prisma Cloud IP addresses and hostname for Code Security to an allow list, to https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.html#id7cb1c15c-a2fa-4072-%20b074-063158eeec08[enable access to the Prisma Cloud Console].
+Integration of Azure Repos with Prisma Cloud console uses a Azure DevOps user token for configuration. You can use the user tokens in two ways:
+
```
Not sure if we fully understand the complexity here in terms of terminology. 
as you mentioned - a single user token can point single organization (by default, just selecting random Azure repos organization if the user has access to multiple ones) or multiple organizations within Azure repos (if the user is part of multiple organizations).

The real improvement we did here is to support **multiple user tokens** that can be integrated in a single prisma cloud tenant) - before this improvement - authorizing a new token override the existing one

vmatangi replied:
@GiladMark why is the support multiple user tokens required? Doesn't the repo get added only once and that is a "single" user's token.? When or why do we need to support multiple user tokens? Is it to allow more than the single user to access the Azure repo directly from our console?

anaghapamidi committed:
Update add-azurerepos.adoc

Update to the introductory content.
@GiladMark does this sound better?

anaghapamidi committed:
updates-based-on-comments

anaghapamidi committed:
merge-to-head

anaghapamidi committed:
updates-based-on-comments

anaghapamidi committed:
resolved-conflicts

anaghapamidi committed:
update-based-on-comments

anaghapamidi committed:
cosmetic-changes

anaghapamidi committed:
cosmetic-changes-2

anaghapamidi committed:
updates-with-review-comments-on-friday

Pubs-MV committed:
Update add-azurerepos.adoc

deleted this sentence "The multiple integrations support ensures new integrations do not override your existing integrations of Azure Repos on Prisma Cloud while ensuring that no additional permission change is required on the Azure DevOps console." 

Other copy edits.

Pubs-MV commented on code-security/admin_guide/get-started/connect-your-repositories/add-azurerepos.adoc:
```diff
@@ -3,37 +3,102 @@
+As a prerequisite, add the Prisma Cloud IP addresses and hostname for Code Security to an allow list, to https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.html#id7cb1c15c-a2fa-4072-%20b074-063158eeec08[enable access to the Prisma Cloud console].
+The integration uses OAuth tokens to help you integrate multiple Azure Repos on the Prisma Cloud console. Enable OAuth tokens on Azure Repos to configure multiple organizations from either the same Azure Repos account or a different one. 
+Currently, support for OAuth user tokens is available only for repositories integrated on Azure Repos. Support for repositories integrated on GitLab or Bitbucket is not yet available.
 
```
Why are we talking about GitLab or Bitbucket in the Azure Repos workflow?

anaghapamidi committed:
update-based-on-comments
